src/Core/WinSWCore/Extensions/AbstractWinSWExtension.cs

```diff
@@ -16,6 +16,11 @@ public virtual void Configure(IWinSWConfiguration descriptor, XmlNode node)
             // Do nothing
         }
 
+        public virtual void Configure(IWinSWConfiguration descriptor, object yamlObject)
+        {
+            // Do nothing
```

Comment:
Throw error about unsupported configuration mode by default? I'd guess the XML support is a bad pattern to follow here